src/patterns/components/masthead/react/code/default.hbs

```diff
@@ -3,6 +3,7 @@
     narrowNavLinks={links.concat(addedNarrowNavLinks)}
     siteLogo={<SiteLogo />}
     utilityContents={utilityItems}
+    navLink={<SprkLink variant="simple" href="#nogo" additionalClasses="sprk-c-Masthead__link">Sign In</SprkLink>}
```

Comment:
Is it called nav-item in vanilla and navLink in react? Why the difference?

Reply:
My thought behind the difference is that nav-item is the class used on the container and this is the actual link that is being passed to be rendered inside the container.